history/provinces/00_k_utgarde.txt

```diff
@@ -301,7 +301,13 @@
 705 = {	# Scalawag Point
 	culture = pirate_culture
 	religion = tidemother
-	holding = tribal_holding
+	holding = castle_holding
+	
+	570.1.1 = {
+		buildings = { 
+			common_tradeport_01
```

Comment:
Can't say I get why you specified it. Does it come from the lore?

Reply:
From [WoWhead](https://wowpedia.fandom.com/wiki/Northsea_Freebooters) : 
> The Northsea Freebooters are a group of pirates that arrived in Howling Fjord and set up a port at Scalawag Point.

It's only a couple of ramshackle buildings but the port at Scalawag Point is (at least in WotLK) the main base of operations for the Northsea Freebooters, so I'd even argue that this is where the capital of the top title should be. Scalawag Point is where the pirates drop off their booty to be traded or shipped elsewhere.

Sorlof's Strand (where the capital currently is) is much more valuable but it's basically a strech of uninhabited beach in WoW and [Captain Ellis](https://wowpedia.fandom.com/wiki/Captain_Ellis) and his crew are sunk and killed by the Sea Giant Sorlof here so that could be reflected by changing the culture of the county is the capital is shifted?

While we are talking about balance, it is a secondary reason for why I added the tradeport building. As discussed in #552, the Southsea Culture is endangered and I figured this would be a small buff for them until #210 is addressed.